tools/rustfmt/srcs/main.rs

```diff
@@ -109,7 +109,7 @@ fn apply_rustfmt(options: &Config, targets: &[String]) {
 
     for target in targets.iter() {
         // Replace any `:` characters and strip leading slashes
-        let target_path = target.replace(":", "/").trim_start_matches('/').to_owned();
+        let target_path = target.replace(':', "/").trim_start_matches('/').to_owned();
```

Comment:
Why these changes??

Reply:
https://github.com/bazelbuild/rules_rust/pull/1153/commits/4ef1f96b94f10131eac25f18d03630683485bb7b - New clippy defects.

Reply:
Aaah I see, thank you!